website/docs/reference/project-collaboration-mode.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Project collaboration mode
+---
+
+:::info Availability
+
+The project collaboration mode is an enterprise-only feature that was introduced in **Unleash 4.22.0**.
+
+:::
+
+A project's **collaboration mode** specifies **who can submit [change requests](change-requests.md)**. There are two collaboration modes:
+- [**open**](#open-collaboration-mode)
+- [**protected**](#protected-collaboration-mode)
+
+## **Open** collaboration mode
+
+Anyone can submit change requests in an **open** project, regardless of their permissions within the project and globally. This is the default collaboration mode for projects.
+
+The open collaboration mode is the default in Unleash and is how all projects worked in Unleash before the introduction of collaboration modes.
+
+## **Protected** collaboration mode
+
+Only admins and project members can submit change requests in a **protected** project. Other users can not submit change requests.
+
+### Change requests
+
+When you change a project's collaboration mode from open to protected, users who do not have the right permissions will lose the ability to create new change requests.
+
+However, existing change requests created by these users will not be deleted. Any users with open change requests will still be able to cancel the change request.
```

Comment:
I'd like to be explicit about these users not being able to update the change request.

```suggestion
However, existing change requests created by these users will not be deleted. Any users with open change requests will still be able to cancel the change request, but they will **not** be able to update the change request.
```